pkg/api/kubegraph/edges.go

```diff
@@ -148,14 +153,18 @@ func AddMountedSecretEdges(g osgraph.Graph, podSpec *kubegraph.PodSpecNode) {
 		syntheticSecret.Name = source.Secret.SecretName
 
 		secretNode := kubegraph.FindOrCreateSyntheticSecretNode(g, syntheticSecret)
+		kind := fmt.Sprintf("%vs", strings.ToLower(secretNode.Kind()))
+		if forbiddenResources.Has(kind) {
+			g.AddEdge(podSpec, secretNode, ForbiddenResourceEdgeKind)
```

Comment:
wasn't expecting an edge.  Can you use the failure list to inform which markers you place?